### ISSUE TITLE
actions set_field pack error

### ISSUE THREAD

**Issue by DanielaSousa** [ljerezchaves/ofswitch13#49]
When the controller sends a Packet_Out with a SetField Action, the buffer_id and other fields would be set to zero, when packing the packet to be sent.
On further debugging, I found that the problem is on function ```ofl_msg_pack_packet_out ``` line: 227 ( https://github.com/ljerezchaves/ofsoftswitch13/blob/625f01384522579db8f1c361209319530d53774a/sources/oflib/ofl-messages-pack.c#L227 )
the pointer to buf is never updated while packing an action, which overwrites the packet_out fields.

**Comment by DanielaSousa** [comment]
Solution: just replace buf with ptr variable

```
ptr += ofl_actions_pack(msg->actions[i], (struct ofp_action_header *)ptr, ptr , exp);
```

**Comment by ljerezchaves** [comment]
Fixed in 43ae19977817d7ecd113ae9603ffb713d5bed3c0
